libs/generators/src/lib/generators/generators.component.scss

```diff
@@ -0,0 +1,42 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  height: calc(100vh - 65px);
+  
+    .container {
+      display: grid;
+      grid-template-columns: 15rem 1fr;
+      grid-template-rows: 1fr;
+      height: 100%;
+    }
+  
+    .side-nav {
+      display: flex;
+      flex-direction: column;
+      background-color: rgb(116, 171, 238);
```

Comment:
in general we will not use custom colors, but this should not be a here after you will use the material side-nav